classes/GlobalDirtEffect.sc

```diff
@@ -15,7 +15,7 @@ GlobalDirtEffect {
 
 	var <>name, <>paramNames, <>numChannels, <state;
 	var <>alwaysRun = false;
-	var synth, defName;
+	var <>synth, defName;
```

Comment:
I think a getter would be enough. One shouldn't really try to make a synth from the outside.

Reply:
Okay this is a valid point and I totally agree! I only need the getter and beside that, I create the global effect synth as usual. I already changed this.